src/python/pants/build_graph/target.py

```diff
@@ -107,15 +107,6 @@ def has_resources(self):
     """
     return hasattr(self, 'resources') and self.resources
 
-  @property
-  def is_exported(self):
-    """Returns True if the target provides an artifact exportable from the repo.
-
-    :API: public
```

Comment:
Hm... I don't think we can just nuke this. Can we apply a hack to deprecate this method while still removing the label? Maybe checking the string value of the classname in the inheritance hierarchy? Alternatively, would it be reasonable to just check for the `provides` property with `getattr`?

Reply:
Gah, I didn't notice it's public API. Will restore it somehow.